pennylane/ops/op_math/prod.py

```diff
@@ -221,20 +169,10 @@ def is_hermitian(self):
         yields false, which ARE hermitian. So a false result only implies a more explicit check
         must be performed.
         """
-        for o1, o2 in combinations(self.factors, r=2):
+        for o1, o2 in combinations(self.operands, r=2):
```

Comment:
We could probably do this in another PR, but this function could now be replaced with:
```suggestion
        return False if self.has_overlapping_wires else all(op.is_hermitian for op in self)
```

Reply:
nice! I'll do it in another PR just in the name of time and simplicity but it's definitely better